src/main/resources/view/MainWindow.fxml

```diff
@@ -26,10 +27,10 @@
       <VBox>
         <MenuBar fx:id="menuBar" VBox.vgrow="NEVER">
           <Menu mnemonicParsing="false" text="File">
-            <MenuItem mnemonicParsing="false" onAction="#handleExit" text="Exit" />
```

Comment:
Looks good, visually its the same as what u showed on discord right?

Haven't analysed too deeply into all the ui files - just wondering, if you remove onAction here, how does the Ui know to handle exit?

Reply:
Sorry, I just tried the actions again and apparently they dont work once they have been removed. These onAction are highlighted in red on Intellij because the code doesn't know if the overall fxml file has a corresponding Ui class. Normally this would be the issue, but the default code given by the tutors actually handles this in the abstract class UiPart, so its alright to have the red highlighted onActions in the code. I've added them back in, which enables the user to clock exit on the File tab on top, and also pop-ups the link to the guides when help is pressed